docs/troubleshooting.md

```diff
@@ -99,6 +99,112 @@ for p in $(oc get pods -l component=es -o jsonpath={.items[*].metadata.name}); d
 done
 ```
 
+## Fluentd
+### Fluentd is holding onto deleted journald files that have been rotated
+This [issue](https://bugzilla.redhat.com/show_bug.cgi?id=1664744) causes `/var/log` to fill up with file handles of 
+deleted journald files.  The only known solution at this time is to periodically cycle Fluentd.  An OKD `CronJob` can be added to perform this function, for example, by checking every 10 minutes the amount of available space to fluentd:
+```
+apiVersion: v1
+kind: Template
+metadata:
+  name: fluentd-reaper
+objects:
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: Role
+  metadata:
+    name: fluentd-reaper
+  rules:
+  - apiGroups:
+    - ""
+    resources:
+    - pods
+    verbs:
+    - delete
+  - apiGroups:
+    - ""
+    resources:
+    - pods/exec
+    verbs:
+    - create
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: RoleBinding
+  metadata:
+    name: fluentd-reaper
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: Role
+    name: fluentd-reaper
+  subjects:
+  - kind: ServiceAccount
+    name: aggregated-logging-fluentd
+    namespace: ${LOGGING_NAMESPACE}
+- apiVersion: batch/v1beta1
+  kind: CronJob
+  metadata:
+    name: fluentd-reaper
+    labels:
+       provider: openshift
+       logging-infra: fluentd-reaper
+  spec:
+    schedule: "${REAP_SCHEDULE}"
+    jobTemplate:
+      spec:
+        template:
+          metadata:
+            labels:
+              provider: openshift
+              logging-infra: fluentd-reaper
+          spec:
+            serviceAccount: aggregated-logging-fluentd
+            serviceAccountName: aggregated-logging-fluentd
+            containers:
+            - env:
+              - name: ENABLED
```

Comment:
remove this env var since its unnecessary

Reply:
removed